CONTRIBUTING.md

```diff
@@ -0,0 +1,586 @@
+
+```sql
+UPDATE planet_osm_line
+  SET mz_road_level = mz_calculate_road_level(highway, railway, aeroway, route, service, aerialway, leisure, sport, man_made, way, name, bicycle, foot, horse, tags->'snowmobile', tags->'ski', osm_id)
+  WHERE
+    highway IN ('pedestrian', 'living_street', 'track', 'path', 'cycleway', 'footway', 'steps') AND
+    way && ST_Transform(ST_SetSrid(ST_MakeBox2D(ST_MakePoint(-124.03564453125, 36.59788913307022), ST_MakePoint(-117.333984375, 39.06184913429154)), 4326), 900913);
+```
+
+### 5. Verify the new logic by running the test
+
+Run the test, hopefully it passes now! You'll need to run the test from the project's root directory, you may need to `cd ../../` to get back there after step 4 above.
+
+```bash
+python integration-test.py local integration-test/875-camp-grounds-zoom.py
+```
+
+**Example output:**
+
+```python
+python integration-test.py local integration-test/875-camp-grounds-zoom.py
+config_url=None
+[   1/1] PASS: 'integration-test/875-camp-grounds-zoom.py'
+PASSED ALL TESTS.
+```
+
+If the test failed like so:
+
+```python
+python integration-test.py local integration-test/875-camp-grounds-zoom.py
+config_url=None
+[   1/1] FAIL: 'integration-test/875-camp-grounds-zoom.py'
+FAILED 1 TESTS. For more information, see 'test.log'
+```
+
+You can investigate why the test failed by printing out the full debug:
+
+```bash
+cat test.log
+```
+
+<div class='alert-message'>NOTE: It's best practice to run your own test AND to also confirm that all other tests are still passing before submitting a pull request. It's possible that you might need to run an overall database migration to achive this locally, or you can rely on CircleCI to run all the tests for you in your branch by pushing it to the server.</div>
+
+#### Some tests require tileserver restart
+
+A minority of issues will require updating the `queries.yaml` file. In those cases you'll also need to restart TileServer to reload this file (and related `jinja` files).
+
+First, switch to the Terminal session where tileserver is running and kill it with a `contrl-c` keyboard press.
+
+Then relaunch tileserver from within the tileserver directory:
+
+```bash
+python tileserver/__init__.py config.yaml
+```
+
+Then run your test starting at the top of step 5 above.
+
+### 6. Perform any modifications, as necessary
+
+Rinse and repeat, rewrite your code. Don't be afraid to ask for help!
+
+### 7. Update data migrations
+
+Once you've finished testing your new database logic in step 4 above you need to record that that same SQL in modified form in `data/migrations/` to ensure someone with an earlier database configuration can catch up with you. (Migrations are reset for each Tilezen release.)
+
+Continuing the `camp_site` example, edit the following in the `data/migrations/v1.0.0-polygon.sql` file:
```

Comment:
Fixed in af86b3626dab5450cc6b513560e0a17ee9070fe1. I added extra text to notify the file name will change based on upcoming release version.